sdk/notificationhubs/notification-hubs/src/api/beginSubmitNotificationHubJob.ts

```diff
@@ -84,9 +87,9 @@ export async function beginSubmitNotificationHubJob(
         }
 
         try {
-          if (!poller.isDone()) {
+          if (!poller.isDone) {
```

Comment:
Let me clarify my understanding for v2 interface first:

- In notification hub SDK we refer the core lro v2 interface but re-implemented the poller in SDK side
- For operation `beginSubmitNotificationHubJob`, the method `submitNotificationHubJob` is to call initial request to trigger the LRO and the `getNotificationHubJob` is to retreive the poller status
- For poller's state, it is initialized with `notStarted` [here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/src/api/beginSubmitNotificationHubJob.ts#L33-L35) and update its value only [poll method](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/src/api/beginSubmitNotificationHubJob.ts#L47).